features/step_definitions/bed_verification_steps.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 Given /^instrument "([^"]*)" has a bed with name "([^"]*)" barcode "([^"]*)" and number (\d+)$/ do |instrument_name, bed_name, bed_barcode, bed_number|
-  instrument = Instrument.find_by_name(instrument_name)
-  bed = Bed.create!(name: bed_name, barcode: bed_barcode, bed_number: bed_number)
+  instrument = Instrument.find_by!(name: instrument_name)
+  bed = Bed.new(name: bed_name, barcode: bed_barcode, bed_number: bed_number)
   instrument.beds << bed
```

Comment:
Rails 5 fix. Beds now require a robot to to changes in the default belongs to options